backend/marina_app/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+# Create your models here.
+
+# WHEN WE MAKE/CHANGE A MODEL, UPDATE:
+#   admin.py
+#   views.py
+#   serializers.py
+#   urls.py
+
+class State(models.Model):
+    name = models.CharField(max_length=200)
+    abbreviation = models.CharField(max_length = 3, default = "")
+    population = models.IntegerField()
+
+    electronic_request = models.BooleanField(default = False)
+    no_notary_required = models.BooleanField(default = False)
+    no_fee = models.BooleanField(default = False)
+    office_contact = models.BooleanField(default = False)
+    no_witness_required = models.BooleanField(default = False)
+
+    def __str__(self):
+        return self.name
+
+class IssueArea(models.Model):
+    title = models.CharField(max_length=200)
+    states = models.ManyToManyField(State)
```

Comment:
what is the benefit of storing a manyToMany relationship as opposed to just storing a list of states?